### PR TITLE
Pre 10

### PR DESCRIPTION
**Pull request by jd-lara**
This is an update to allow PSY 1.0 to parse RTS-GMLC data. 

@claytonpbarrows there are several issues with the HydroData here that PSY 1.0 is picking up. Namely, several HydroDispatch devices have `get_storage_capacity` as the multiplier.

### CODE REVIEW

**Comment by jeanpaulwatson** [comment]
Can you comment on / summarize the diffs in gen.csv? At least these seem to extend beyond hydro. I'm specifically worried about (1) regressions and (2) existing translators to non-SIIP PCMs.

File: RTS_Data/FormattedData/SIIP/user_descriptors.yaml [hunk 1]
```diff
@@ -70,10 +70,21 @@ generator:
 - {custom_name: Output_pct_2, name: output_point_2, unit_system: device_base}
 - {custom_name: Output_pct_3, name: output_point_3, unit_system: device_base}
 - {custom_name: Output_pct_4, name: output_point_4, unit_system: device_base}
-- {custom_name: Base MVA, name: base_mva}
+- {custom_name: MATPOWER BaseMVA, name: base_mva} #TODO just for testing
```
Comment (claytonpbarrows) [inline RTS_Data/FormattedData/SIIP/user_descriptors.yaml:73]: remove this line

**Review by claytonpbarrows — COMMENTED** [review]
I don't think we need the changes from gen.csv and you can remove the one line that I commented on in the json.

**Comment by jeanpaulwatson** [comment]
Cool - what about the non-SIIP timeseries pointer file changes?

File: RTS_Data/FormattedData/SIIP/user_descriptors.yaml (hunk 1 above)
Comment (jd-lara) [reply RTS_Data/FormattedData/SIIP/user_descriptors.yaml]: solved via  33fb51f

**Comment by jd-lara** [comment]
> Cool - what about the non-SIIP time-series pointer file changes?

I removed those changes in 2d6001d to avoid issues and made a note that these pointers won't work for SIIP without some additional changes.

File: RTS_Data/FormattedData/SIIP/user_descriptors.yaml (hunk 1 above)
Comment (jd-lara) [reply RTS_Data/FormattedData/SIIP/user_descriptors.yaml]: The previous line `- {custom_name: Base MVA, name: base_mva}` needs to be added back.

**Pull request merged**